firmware/controllers/can/can_dash.cpp

```diff
@@ -155,7 +155,8 @@ void canMazdaRX8(CanCycle cycle) {
 
 			float kph = getVehicleSpeed();
 
-			msg.setShortValue(SWAP_UINT16(GET_RPM() * 4), 0);
+			int rpm = GET_RPM();
```

Comment:
i do not understand if this changes everything?
we need code comment or maybe even automated testing coverage if that change affects the outcome somehow

Reply:
I've cherry-picked with a comment please rebase